libs/init/migrate/src/mill/init/migrate/PackageWriter.scala

```diff
@@ -0,0 +1,334 @@
+package mill.init.migrate
+
+import mill.constants.CodeGenConstants.rootModuleAlias
+import mill.internal.Util.backtickWrap
+import pprint.Util.literalize
+
+import java.io.PrintStream
+import scala.collection.immutable.SortedSet
+
+trait PackageWriter {
```

Comment:
We should re-use as much of `BuildGenUtil` as possible here, which already has logic for rendering `def mvnDeps`, `def moduleDeps`, etc. We should not need to re-implement it from scratch and end up maintaining two different implementations

Reply:
I was hoping to port the other importers to the new design. This might eliminate `BuildGenUtil` entirely.
Can we discuss this?